cfgov/legacy/templates/front/base_nonresponsive.html

```diff
@@ -61,8 +61,9 @@
     <link rel="apple-touch-icon" size="114x114" href="{% static 'nemo/_/img/apple-touch-icon-114x114-precomposed.png' %}">
     <link rel="apple-touch-icon" size="144x144" href="{% static 'nemo/_/img/apple-touch-icon-144x144-precomposed.png' %}">
 
+    {% block jquery %}
```

Comment:
Can this empty block be removed?

Reply:
I think it's needed to tell the templates that extend this template where to place jquery. I would have placed jquery in the `app_js` block, but it was originally in the head so I aimed to keep it in the same place.

Reply:
Ahh [indeed](https://docs.djangoproject.com/en/1.7/topics/templates/#id1)!